### DISCUSSION TITLE
Returning auth.users' when after signInWithOtp().

### DISCUSSION THREAD

**Discussion by andrelaurent1** [supabase/supabase#18424] [Questions]
I'm trying to get the ID of the user immediately after the user has been created. but the response on the call is a blank JSON body. is there any way for me to get the newly created user ID?

![image](https://github.com/supabase/supabase/assets/106798934/8303f577-7f69-4192-90c1-95de32836edb)

**Comment by GaryAustin1** [comment]
signInWithOtp redirects back to your app when the user clicks on the email.  There is no user session or data until then.  The return data should be null.
